pom.xml

```diff
@@ -569,6 +569,49 @@
                 <artifactId>h2o</artifactId>
                 <version>${h2o.version}</version>
             </dependency>
+
+            <!--DL4J-->
+            <dependency>
+                <groupId>org.deeplearning4j</groupId>
+                <artifactId>deeplearning4j-core</artifactId>
+                <version>${dl4j.core.version}</version>
+            </dependency>
+
+            <!-- ND4J-->
+            <!-- https://mvnrepository.com/artifact/org.nd4j/nd4j-api -->
+            <dependency>
+                <groupId>org.nd4j</groupId>
+                <artifactId>nd4j-api</artifactId>
+                <version>0.4.0</version>
```

Comment:
Define the version under the <properties> tag. Refer other dependencies for example.